api/src/main/resources/workbench-api.yaml

```diff
@@ -6017,6 +6141,20 @@ definitions:
     - Deleting
     - Deleted
     - Unknown
+  AppStatus:
+    type: string
+    enum:
+      - STATUS_UNSPECIFIED
+      - RUNNING
+      - ERROR
+      - DELETING
+      - DELETED
+      - PROVISIONING
```

Comment:
Is there a STOPPED status?

Reply:
good point. It is not supported? https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/swagger/api-docs.yaml#L2424L2432
I will ask IA team

Reply:
https://github.com/DataBiosphere/leonardo/pull/2888 that is fixed!